tests/unit/manage/views/test_organizations.py

```diff
@@ -1590,11 +1590,16 @@ def test_add_organization_project_new_project(
         view = org_views.ManageOrganizationProjectsViews(organization, db_request)
         result = view.add_organization_project()
 
-        # The project was created, and belongs to the organization.
+        # The project was created
         project = (
             db_request.db.query(Project).filter_by(name="fakepackage").one_or_none()
         )
         assert project is not None
```

Comment:
If we now expect that the project can never be None, we can remove one assert and update this assignment.

```suggestion
        project = (
            db_request.db.query(Project).filter_by(name="fakepackage").one()
        )
```